Content/ReleaseBriefings/4_6/Section_BreakingChanges.tex

```diff
@@ -0,0 +1,90 @@
+%==========================================================================
+
+\begin{frame}[fragile]
+
+  {\Huge Deprecations and other breaking changes}
+
+  \vspace{10pt}
+
+\end{frame}
+
+
+\begin{frame}[fragile]{Intel Classic Compiler}
+  \begin{itemize}
+    \item Intel has long since deprecated Intel Classic (since around 2022), and removed from oneAPI 2024.0 release
+    \item In order to focus on newer compilers, and reduce maintenance burden, we have \textbf{removed} support for Intel Classic (oneAPI Intel/icpx still supported of course!)
+  \end{itemize}
+\end{frame}
+
+
+\begin{frame}[fragile]{DualView changes}
+  \textbf{Deprecate} direct access to \texttt{d\_view} and \texttt{h\_view}
+  \begin{itemize}
+    \item Modifying the allocations in d\_view and h\_view directly is dangerous, especially if \texttt{modify} and \texttt{sync} are skipped
+    \item Use \texttt{view\_host()} and \texttt{view\_device()} instead
+    \item These two functions return by value with deprecated code enabled and by const reference otherwise. This might have perfomance implications if used extensively, e.g., in loop bounds.
+  \end{itemize}
+\end{frame}
+
+
+\begin{frame}[fragile]{Experimental SIMD changes}
+  \begin{itemize}
+    \item \texttt{native\_simd}, \texttt{native\_simd\_mask} \textbf{deprecated} to align with the C++26 standard
+    \item \textbf{Removed} Obtaining a reference from \texttt{*simd*::operator[]} to align with the C++26 Standard
+    \item \textbf{Changed} the return type of \texttt{Kokkos::Experimental::*simd*::operator==} and \texttt{operator!=} to return SIMD masks instead of \texttt{bool}
```

Comment:
```suggestion
    \item \textbf{Removed} Obtaining a reference from SIMD \texttt{operator[]} to align with the C++26 Standard
    \item \textbf{Changed} the return type of SIMD \texttt{operator==} and \texttt{operator!=} to return SIMD masks instead of \texttt{bool}
```

This is to remove confusion between the types that have these operations (e.g. `simd, `basic_simd`, `simd_mask`, `basic_simd_mask`